packages/tsconfig/base.json

```diff
@@ -2,20 +2,21 @@
   "$schema": "https://json.schemastore.org/tsconfig",
   "compilerOptions": {
     "target": "es5",
-    "module": "esnext",
-    "lib": ["dom", "esnext", "esnext.intl", "es2017.intl", "es2018.intl"],
+    "module": "ES2020",
```

Comment:
This seems like a pretty big change.  What's up?

Reply:
I can change it back to `esnext` but that is a floating value so I locked it down to make troubleshooting a bit easier. If we do always want access to the latest specs I can change it back.

This honestly only got committed by mistake but I left it since everything was working with it set and the next module was making things harder to pin down.

Reply:
Oh, OK.  Yeah, I'm all for pinning down floating values.  Let's do that.